cap-ci/scripts/destroy_k8s.tmpl

```diff
@@ -9,5 +9,14 @@ if [[ ${BACKEND} == "gke" ]]; then
     export GKE_CRED_JSON=$PWD/gke-key.json
     export SCF_OPERATOR=true
     make clean
+elif [[ ${BACKEND} == "aks" ]]; then
+    printf "%s" '((azure-dns-json))' > $PWD/azure-dns-key.json
+    export AZURE_DNS_JSON=$PWD/azure-dns-key.json
+    export AZURE_APP_ID='((azure-sp-creds.app_id))'
+    export AZURE_PASSWORD='((azure-sp-creds.password))'
+    export AZURE_TENANT_ID='((azure-sp-creds.tenant_id))'
+    export AZURE_SUBSCRIPTION_ID='((azure-sp-creds.subscription_id))'
+    export SCF_OPERATOR=true
```

Comment:
The SCF_OPERATOR=true is not needed now (https://github.com/SUSE/catapult/blob/master/modules/kubecf/clean.sh now has conditionals and checks for everything). The idea is that deploying a cluster is not tied to whatever kubecf flavour, as we make sure that way that kubecf works on as broad cluster types as possible.

```suggestion

```